src/app/templates/partials/metadata-tab.ftl

```diff
@@ -0,0 +1,6 @@
+<#-- @ftlvariable name="report" type="org.vaccineimpact.orderlyweb.models.ReportVersionDetail" -->
+
+<#include "report-title.ftl">
+<div class="container ml-0">
```

Comment:
I see you have tested this above in the versionPageTests, but probably best to add its own test file with a test that this is rendered.

Reply:
Content for all the tabs is currently tested in VersionPageTests. You're suggesting splitting that up by tab? It is pretty long and it's going to get longer with all the new metadata so that's a reasonable refactor

Reply:
Yeah, I just figured it would have to be done eventually.